packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/clusters.routing.js

```diff
@@ -25,6 +30,91 @@ export default /* @ngInject */ ($stateProvider) => {
         squash: true,
       },
     },
-    resolve: {},
+    resolve: {
+      filter: /* @ngInject */ ($transition$) => $transition$.params().filter,
+      orderUrl: /* @ngInject */ (User) => User.getUrlOf('dedicatedOrder'),
+      getClusterDashboardLink: /* @ngInject */ ($state) => ({ iam }) =>
+        $state.href('app.dedicated-cluster.cluster', {
+          clusterId: iam.displayName,
+        }),
+      dedicatedClusters: /* @ngInject */ ($transition$, iceberg) => {
+        const { filter, pageSize, sort, sortOrder } = $transition$.params();
+        let { page } = $transition$.params();
+        const filtersParsed = JSON.parse(filter);
+
+        if (filtersParsed.length > 0) {
+          page = 1;
+        }
+
+        let request = iceberg('/dedicated/cluster')
+          .query()
+          .expand('CachedObjectList-Pages')
+          .limit(pageSize)
+          .offset(page)
+          .sort(sort, sortOrder);
+
+        const FILTER_OPERATORS = {
+          contains: 'like',
+          is: 'eq',
+          isAfter: 'gt',
+          isBefore: 'lt',
+          isNot: 'ne',
+          smaller: 'lt',
+          bigger: 'gt',
+          startsWith: 'like',
+          endsWith: 'like',
+        };
```

Comment:
Maybe we can reuse directly the constant to layout helper module ?

Reply:
with import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
you can use for avoid big diplicate: 

```suggestion
parsedFilters.forEach(({ field, comparator, reference }) => {
            request = request.addFilter(
              field,
              get(ListLayoutHelper.FILTER_OPERATORS, comparator),
              ListLayoutHelper.mapFilterForIceberg(comparator, reference),
            );
          });
          ```